src/alire/alire-publish.adb

```diff
@@ -187,9 +187,15 @@ package body Alire.Publish is
 
       Index_On_Disk.Loading.Load_All (Strict => True).Assert;
       if Index.Exists (Release.Name, Release.Version) then
-         Raise_Checked_Error
-           ("Target release " & Release.Milestone.TTY_Image
-            & " already exist in a loaded index");
+         if not Force then
+            Raise_Checked_Error
+               ("Target release " & Release.Milestone.TTY_Image
+                & " already exist in a loaded index");
+         end if;
+         Ada.Text_IO.New_Line;
+         Trace.Warning
+            ("Target release " & Release.Milestone.TTY_Image
+             & " already exist in a loaded index (ignored)");
```

Comment:
Thanks for the PR, @stcarrez. You can better use here `Alire.Recoverable_Error`, which does essentially the same with some other minor tweaks and is a drop-in replacement for `Raise_Checked_Error`.